README.rst

```diff
@@ -5,23 +5,21 @@ Ansible
 *******
 
 Ansible is a radically simple IT automation system. It handles
-configuration-management, application deployment, cloud provisioning,
-ad-hoc task-execution, and multinode orchestration -- including
-trivializing things like zero-downtime rolling updates with load
-balancers.
+configuration management, application deployment, cloud provisioning,
```

Comment:
How about we include network automation and maybe even devops here as well? After all, network automation is a huge topic with Ansible, and many who read "configuration management" will fail to see that this includes network devices.

Reply:
Addressed by 1f59d0c0118c11a56cb1a4d760b176290287e12e.